src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/templates/fields/Select.tsx

```diff
@@ -0,0 +1,28 @@
+import type {SelectFieldProps} from '@givewp/forms/propTypes';
+
+export default function Select({label, placeholder, fieldError, options, inputProps}: SelectFieldProps) {
+    return (
+        <label>
+            <span>{label}</span>
+
+            <select  {...inputProps} aria-invalid={fieldError ? 'true' : 'false'}>
+                {placeholder ? (
+                    <>
+                        <option hidden>{placeholder}</option>
+                        <option disabled>{placeholder}</option>
+                    </>
+                ) : <option disabled>{' '}</option>
```

Comment:
I think it should be up to the Select instance whether or not it has a blank option. Some selects want to force that there always is a value, which is basically the default of the select UX.

Reply:
👍 2b053927

Also, apparently setting default values when they are null was causing some side effects... noticeable in the select field because nothing was showing up.  Fixed that here fd04b300 and now the select field is back to it's native functionality.